### DISCUSSION TITLE
Which NuGet feed should I use to get Microsoft.NETCore.App.Ref version 6.0.34

### DISCUSSION THREAD

**Discussion by prochan2** [dotnet/source-build#4546] [Q&A]
I use daily build of .NET SDK. Building a project targeting .NET 6 fails on

```
error NU1102: Unable to find package Microsoft.NETCore.App.Ref with version (= 6.0.34)
```

Is there a NuGet feed that provides this version?

**Comment by MichaelSimons** [comment]
This doesn't appear to be source-build related.  Moving to the SDK discussions.

**Reply by MichaelSimons** [reply]
Looks like I can't transfer.  @prochan2 - please post this question at https://github.com/dotnet/sdk/issues.

**Reply by prochan2** [reply]
Thanks, posted: https://github.com/dotnet/sdk/issues/42638
